app/views/podcast_episodes/_sidebar.html.erb

```diff
@@ -9,23 +9,23 @@
         <div class="widget-body">
           <% @podcasts.each do |podcast| %>
             <a href="/<%= podcast.slug %>">
-              <span class="podcast-pic" id="podcast-pic-<%=podcast.slug %>">
+              <div class="podcast-pic" id="podcast-pic-<%= podcast.slug %>">
```

Comment:
This change might cause unintended CSS issue. Can you please revert this to `span`?

Reply:
@maestromac a span element can not contain a div element.

Reply:
Hmmm good point. I've toyed around and it doesn't seem to break anything.